src/skins/vector/css/matrix-react-sdk/views/messages/TextualEvent.css

```diff
@@ -15,6 +15,11 @@ limitations under the License.
 */
 
 .mx_TextualEvent {
-    opacity: 0.5;
-    overflow-y: hidden;
+    font-weight: 600;
```

Comment:
designwise i'm slightly surprised that we're making these bolder rather than lighter, as typically they are less important than the actual text messages on the page.  This contradicts @antikewl's earlier designs, and i can't see anything in zeplin that suggests they should be like this.  What is the rationale? :)